app/models/radio_facet.rb

```diff
@@ -1,4 +1,8 @@
 class RadioFacet < FilterableFacet
+  def value
+    @value || default_value['value']
```

Comment:
Where is `@value` defined? There's a `value` on the `FilterableFacet` base class, but I don't think it's this?

Reply:
It's quite hard to see. It's set within `FacetCollection` here:

https://github.com/alphagov/finder-frontend/blob/master/app/models/facet_collection.rb#L15

This gets called in the `FinderPresenter`:

https://github.com/alphagov/finder-frontend/blob/master/app/presenters/finder_presenter.rb#L16

This is how values from the params are translated into their corresponding facets.